docs/fundamentals/dashboard/standalone-for-nodejs.md

```diff
@@ -0,0 +1,210 @@
+
+    ```console
+    npm install @opentelemetry/api @opentelemetry/sdk-node @opentelemetry/exporter-trace-otlp-grpc @opentelemetry/exporter-metrics-otlp-grpc @opentelemetry/auto-instrumentations-node
+    ```
+
+1. Create a new file called `tracing.js` to configure OpenTelemetry:
+
+    :::code language="javascript" source="~/docs/fundamentals/dashboard/snippets/standalone-for-nodejs/tracing.js":::
+
+1. Update your `app.js` to import the tracing configuration at the very beginning:
+
+    ```javascript
+    // This must be imported first!
+    require('./tracing');
+
+    const express = require('express');
+    const app = express();
+    const port = process.env.PORT || 3000;
+
+    app.get('/', (req, res) => {
+        console.log('Received request for home page');
+        res.json({
+            message: 'Hello from Node.js!',
+            timestamp: new Date().toISOString()
+        });
+    });
+
+    app.get('/api/weather', (req, res) => {
+        const weather = [
+            { city: 'Seattle', temperature: 72, condition: 'Cloudy' },
+            { city: 'Portland', temperature: 68, condition: 'Rainy' },
+            { city: 'San Francisco', temperature: 65, condition: 'Foggy' },
+            { city: 'Los Angeles', temperature: 78, condition: 'Sunny' }
+        ];
+        
+        console.log('Received request for weather data');
+        res.json(weather);
+    });
+
+    app.listen(port, () => {
+        console.log(`Server running at http://localhost:${port}`);
+    });
+    ```
+
+1. Restart your application:
+
+    ```console
+    npm start
+    ```
+
+## Start the Aspire dashboard
+
+To start the Aspire dashboard in standalone mode, run the following Docker command:
+
+### [Bash](#tab/bash)
+
+```bash
+docker run --rm -it -p 18888:18888 -p 4317:18889 --name aspire-dashboard \
+    mcr.microsoft.com/dotnet/aspire-dashboard:9.0
+```
+
+### [PowerShell](#tab/powershell)
+
+```powershell
+docker run --rm -it -p 18888:18888 -p 4317:18889 --name aspire-dashboard `
+    mcr.microsoft.com/dotnet/aspire-dashboard:9.0
+```
+
+---
+
+In the Docker logs, the endpoint and key for the dashboard are displayed. Copy the key and navigate to `http://localhost:18888` in a web browser. Enter the key to log in to the dashboard.
+
+## View telemetry in the dashboard
+
+After starting both the dashboard and your Node.js application, you can view telemetry data by making requests to your application and observing the results in the dashboard.
+
+1. Make some requests to your Node.js application:
+
+    ```bash
+    curl http://localhost:3000/
+    curl http://localhost:3000/api/weather
+    ```
+
+1. Navigate to the .NET Aspire dashboard at `http://localhost:18888` and explore the different sections:
+
+### Structured logs
+
+The **Structured Logs** page displays logs from your Node.js application. You'll see console log entries and HTTP request logs automatically captured by the OpenTelemetry auto-instrumentation.
```

Comment:
I'm not sure why, but there are no structured logs for this app setup. There are, however; both traces and metrics.